cowprotocol/cowswap
# feat: use cow widget lib

# Summary

1. Added `@cowprotocol/widget-lib` to init CowSwap widget
2. Added `enabledTradeTypes` to control which trade widgets are enabled
3. Added a control to the configurator to control the current trade type

  # To Test

1. Should work the same as before
2. "Current trade type" control should change the current trade widget inside of the iframe
